ansible/system.yml

```diff
@@ -57,54 +64,56 @@
   become: yes
   roles:
     - role: postgresql
-      when: esrender_db_type == 'postgresql'
+      when: esrender_db_type == 'postgresql' and edu_version is version('7.0.0','<')
     - role: mariadb
-      when: esrender_db_type == 'mariadb'
+      when: esrender_db_type == 'mariadb' and edu_version is version('7.0.0','<')
     - renderingservice-installation
     - role: splash
-      when: install_splash is defined and install_splash
+      when: install_splash is defined and install_splash and edu_version is version('7.0.0','<')
```

Comment:
It isn't userfriendly if the user sets `install_splash: true` and it just don't installs it. Should also be possible to install splash, if the user wants to.